component/main.jsonnet

```
@@ -77,6 +77,8 @@ local consoleSpec =
     [k]: params.config[k]
     for k in std.objectFields(params.config)
     if k != 'route'
+  } + {
+    [if !std.objectHas(params.config, 'plugins') then 'plugins']: [],
```

I'd probably just do this as

```suggestion
  } + {
    plugins+: [],
```

which should have the same effect

---

So easy 😅